Lib/test/test_import/__init__.py

```diff
@@ -27,6 +27,10 @@
     TESTFN, rmtree, temp_umask, TESTFN_UNENCODABLE, temp_dir)
 from test.support import script_helper
 from test.support import threading_helper
+from test.support.import_helper import (forget, make_legacy_pyc, unlink,
+                                        unload, DirsOnSysPath)
+from test.support.os_helper import (TESTFN, rmtree, temp_umask,
+                                    TESTFN_UNENCODABLE, temp_dir)
```

Comment:
It seems like these changes are already merged into master via the GH-21412: commit 96a6a6d42be272a27562d98549bbffc0d1854669. Maybe rebase your PR?

Reply:
Oh, sorry for this terrible rebase operation, I use a new PR to replace it.